dagger/ci/cli-tests/00-account-deposit.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+recall --network localnet account deposit 1
```

Comment:
Maybe the execution script in `ci/main.go` can set envvar `RECALL_NETWORK=localnet` so that the scripts can assume the correct environment and there is no need to pall `--network localnet` for each invocation?

Reply:
I think it would be great to add a simple assertion just to show how it can be done.